src/io/ResultSet.jl

```diff
@@ -212,6 +212,7 @@ function combine_results(result_sets...)::ResultSet
                 nrow(all_inputs),
                 size(rs1.seed_log, :timesteps),
                 size(rs1.seed_log, :sites),
+                36 # TODO: Find a way to remove this constant
```

Comment:
Oops, this may have been a defunct change that crept in.

Reply:
I only see one method for this function `setup_logs` and it sccepts 5 arguments: `setup_logs(z_store, unique_sites, n_scens, tf, n_sites)`. I think this line should be removed, no?

Reply:
Yeah I'm trying to work out how it crept in...